packages/react-scripts/template/src/Components/Landing.js

```diff
@@ -4,7 +4,10 @@ export default class Landing extends React.Component {
   render() {
     return (
       <div className="landing">
-        <p>‍👨‍🚀</p>
+        <section className="easter-egg">
+          <p className="easter-egg__spacheman">👨‍🚀</p>
+          <p className="easter-egg__rocket">🚀</p>
+        </section>
```

Comment:
rename `easter-egg` to `outerspace` , `easter-egg__spacheman` to `spacheman` and `easter-egg__rocket` to `rocket`